Makefile.sp

```diff
@@ -4,17 +4,19 @@
 # You can set these variables from the command line, and also
 # from the environment for the first two.
 SPHINXDIR     = .sphinx
-SPHINXOPTS    ?= -c . -d $(SPHINXDIR)/.doctrees
+SPHINXOPTS    ?= -c . -d $(SPHINXDIR)/.doctrees -j auto
 SPHINXBUILD   ?= sphinx-build
 SOURCEDIR     = .
 BUILDDIR      = _build
 VENVDIR       = $(SPHINXDIR)/venv
 PA11Y         = $(SPHINXDIR)/node_modules/pa11y/bin/pa11y.js --config $(SPHINXDIR)/pa11y.json
 VENV          = $(VENVDIR)/bin/activate
 TARGET        = *
+ALLFILES      =  *.rst **/*.rst
```

Comment:
It does, that's why we need this. By moving the list of files to a variable it's possible to override it when executing the target.
See here for an example: https://github.com/canonical/lxd/blob/0015ee92a9c3ce2f629281ac89c25ee3cee10a48/doc/Makefile#L54

Reply:
It probably should - but I think we can leave that for https://warthogs.atlassian.net/browse/DOCPR-525 instead of doing the work twice. It's not a regression, you just couldn't run the woke target on Markdown files at all before. ;)